src/geo/ui/widgets/widgets_collection.js

```diff
@@ -0,0 +1,41 @@
+var Backbone = require('backbone');
+var _ = require('underscore');
+
+/**
+ *  Collection that controls widget models per layer
+ *
+ */
+
+module.exports = Backbone.Collection.extend({
+
+  initialize: function() {
+    this._initBinds();
+  },
+
+  _initBinds: function() {
+    // If a category model applies the category colors, rest should remove/disable
+    // the category colors applied before.
+    this.bind('change:categoryColors', function(m, isColorCategorized) {
```

Comment:
Since we have the model and we know the name of the attribute that changed, we could get rid of `isColorCategorized` and use `m.get('wadus')` instead?